plutus-ledger-api/src/PlutusLedgerApi/Common/ProtocolVersions.hs

```diff
@@ -86,16 +94,26 @@ knownPVs =
     , valentinePV
     , changPV
     , plominPV
+    , pv11PV
     ]
 
--- | This is a placeholder for when we don't yet know what protocol version will
--- be used for something. It's a very high protocol version that should never
--- appear in reality.  New builtins should always be given this protocol version
--- until they've been finalised.
---
--- We should not assign names to future protocol versions until it's
--- confirmed that they are correct, otherwise we could accidentally
--- associate something with the wrong protocol version.
+-- We're sometimes in an intermediate state where we've added new builtins but
+-- not yet released them (but intend to). This is used by some of the tests to
+-- decide what PVs the test should include.  UPDATE THIS when we're expecting to
```

Comment:
Rather than saying "UPDATE THIS" in a comment here - which is extremely easy to overlook - it would be nice to have a checklist somewhere, that documents everything we need to do when we are ready to release new builtins.

Reply:
Ah, I have an [issue](https://github.com/IntersectMBO/plutus-private/issues/1495) to do that: thanks for reminding me. I'll put the issue in the current sprint so I can do it while the process is still fresh in my mind.   I had in mind to write some kind of textual document, but maybe an issue template would be a better way to do it.